### PR TITLE
Admedia adapter

### PR DESCRIPTION
**Pull request by Elardzhi**
## Type of change

- [x ] New bidder adapter   

## Description of change
Admedia bid adapter

- test parameters for validating bids
```
{
  bidder: 'admedia',
  params: {
    aid: 86858
  }
}
```

### CODE REVIEW

**Comment by bretg** [comment]
docs PR https://github.com/prebid/prebid.github.io/pull/1116

File: modules/admediaBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import * as utils from '../src/utils';
+import {registerBidder} from '../src/adapters/bidderFactory';
+// import { config } from 'src/config';
```
Comment (jaiminpanchal27) [inline modules/admediaBidAdapter.js]: Please remove if not needed

**Comment by jaiminpanchal27** [comment]
@Elardzhi

File: modules/admediaBidAdapter.js (hunk 1 above)
Comment (Elardzhi) [reply modules/admediaBidAdapter.js]: Removed

**Pull request merged**